powerai_serverside/datasequence.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import tensorflow as tf
```

Comment:
imports should be alphabetized and groups by standard lib and 3rd party...

```
import math
import os
import random

from keras.preprocessing.image import img_to_array
from keras.preprocessing.image import load_img
from keras.utils import Sequence
import numpy as np
import tensorflow as tf
```

Reply:
Also, don't import classes `from keras.utils import Sequence`, use `from keras import utils` and use the module.